### PR TITLE
Added styled call to action button

### PR DESCRIPTION
**Pull request by alecbar**
For #2 - pretty basic styled button with larger font size

Before:
Screen Shot 2020-05-26 at 1 48 51 PM

After:
Screen Shot 2020-05-26 at 1 48 58 PM

Let me know if there are any other improvements or changes I can make!

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -204,6 +202,21 @@ export default function Home() {
           height: 1em;
         }
 
+        .button-primary {
+            color: white;
```
Comment (phil-ociraptor) [inline pages/index.js:206]: nit: can you make the color ##212121 ? (or the equivalent in rgba)

File: pages/index.js [hunk 2]
```diff
@@ -204,6 +202,21 @@ export default function Home() {
...
+            background-color: rgba(77,208,225, 1);
+            border-radius: 10px;
+            padding: 10px;
+            font-size: 1.5rem;
+        }
+
+        .button-primary:active,
+        .button-primary:focus,
+        .button-primary:hover {
+          background-color: rgba(77,208,225, .75);
```
Comment (phil-ociraptor) [inline pages/index.js:216]: good solution for now, but something we may want a designer (i.e. Van) to take a second look at and chime in how we should deal with the hover interaction

File: pages/index.js [hunk 3]
```diff
@@ -204,6 +202,21 @@ export default function Home() {
           height: 1em;
         }
 
+        .button-primary {
+            color: (33, 33, 33);
```
Comment (phil-ociraptor) [inline pages/index.js:206]: this should be: `color: rgb(33, 33, 33);` (right now the `rgb` is missing)

or to keep consistent with the other color values, `color: rgba(33, 33, 33, 1);`

Comment (alecbar) [reply pages/index.js]: Whoops - easy fix. Should be good now.

**Comment by phil-ociraptor** [comment]
Looks good!

**Comment by alecbar** [comment]
Cool, thanks for helping me make my first pull request and merge happen!

**Pull request merged**